### ISSUE TITLE
Cannot add Wordpress blog on safari

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#565]
```
What's the problem?

I tried to add a wordpress blog, but after I click the authorize to scribefire 
button on wordpress, I saw scribe fire reopened without any blog account added 
into it.


What browser are you using?

Safari 5.1.2

What version of ScribeFire are you running?

4scrscribefire
scscc
```

-----
Original issue reported on code.google.com by `zju...@gmail.com` on 2 Feb 2012 at 6:43

**Comment by GoogleCodeExporter** [comment]
```
I am having this problem as well. Anybody have any ideas
```

-----
Original comment by `j...@thecurrins.com` on 11 Dec 2012 at 10:29

**Comment by GoogleCodeExporter** [comment]
```
Fixed in ScribeFire version 4.2 for Safari
```

-----
Original comment by `scribefi...@gmail.com` on 3 Sep 2013 at 9:37
* Changed state: **Fixed**
